### PR TITLE
remove UAI

### PR DESCRIPTION
**Pull request by hawestra**
remove UAI token manager from GSQ compute histogram component and use run token instead
pin to api version 12-01-2022

success run : https://ml.azure.com/experiments/id/1a0a9a51-a26e-44b3-92d2-dcfd2d1fe6aa/runs/cool_raisin_yvdgxt2tzk?wsid=/subscriptions/e0fd569c-e34a-4249-8c24-e8d723c7f054/resourceGroups/hawestra-rg/providers/Microsoft.MachineLearningServices/workspaces/hawestra-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

### CODE REVIEW

File: assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py [hunk 1]
```diff
@@ -755,31 +755,7 @@ def get_token(self):
...
-            or time.time() - self.last_refresh_time > AZURE_TOKEN_REFRESH_INTERVAL
-        ):
-            self.last_refresh_time = time.time()
-            self.token = self.credential.get_token(self.token_scope.value).token
-
-        return self.token
-
-
-class _UAIAPITokenManager(_APITokenManager):
+class _WorkspaceConnectionTokenManager(_APITokenManager):
```
Comment (yunjie-hub) [inline assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py:758]: Do we need to inherit from _APITokenManager

File: assets/model_monitoring/components/generation_safety_quality/generation_safety_quality_signal_monitor/spec.yaml [hunk 2]
```diff
@@ -137,7 +137,7 @@ jobs:
       type: managed
   compute_metrics:
     type: spark
-    component: azureml://registries/azureml/components/gsq_annotation_compute_metrics/versions/0.4.0
+    component: azureml://registries/azureml/components/gsq_annotation_compute_metrics/versions/0.4.1
```
Comment (yunjie-hub) [inline assets/model_monitoring/components/generation_safety_quality/generation_safety_quality_signal_monitor/spec.yaml:140]: do we need to bump this version?

Comment (yunjie-hub) [reply assets/model_monitoring/components/generation_safety_quality/generation_safety_quality_signal_monitor/spec.yaml:140]: Oh, I find you update the yaml

**Review by yunjie-hub — DISMISSED** [review]

File: assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py (hunk 1 above)
Comment (hawestra) [reply assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py:758]: no, I will clean this up in a follow up PR

**Review by yunjie-hub — APPROVED** [review]

File: assets/model_monitoring/components/generation_safety_quality/annotation_compute_histogram/spec.yaml
Comment (yunjie-hub) [inline assets/model_monitoring/components/generation_safety_quality/annotation_compute_histogram/spec.yaml:1]: Identity needs to change to AmlToken then we don't need UAI any more

**Pull request merged**